aiohttp/server.py

```diff
@@ -110,8 +108,7 @@ def __init__(self, *, loop=None,
         self.port = port
         self.logger = log or logger
         self.debug = debug
-        self.access_log = access_log
```

Comment:
Please keep `.access_log` and `.access_log_format` -- they are part of public API